src/session/sidebar/chat_row.rs

```diff
@@ -358,6 +359,43 @@ fn stringify_message(message: Message) -> String {
                 format!(", {}", dim_and_escape(&data.caption.text))
             }
         ),
+        MessageContent::MessageCall(data) => {
+            match data.discard_reason {
+                CallDiscardReason::Declined => {
+                    if message.is_outgoing() {
+                        // Telegram Desktop/Android labels declined outgoing calls just as
+                        // "Outgoing call" and puts a red arrow in the message bubble. We should be
+                        // more accurate here.
+                        if data.is_video {
+                            gettext("Unanswered video call")
```

Comment:
Looking at the docs it seems that here the call should have been declined by the other party, so I guess we should still label this as `declined`? In that case we should also label if the call was outgoing or not.

Reply:
I changed it in the way you proposed. Feel free to reopen if I have not understood correctly.